### ISSUE TITLE
[TIL]김현재 201223

### ISSUE THREAD

**Issue by journey101** [codestates/ds-TIL#1249]
- **키워드**: 
> 학습 규제 전략
> droput
> batchsize, epoch
> weight decay 

- **배운 것**:
> keras에서 학습을 규제(오버피팅을 방지하는)하는 법
> 역전파 복습

- **어려웠던 부분/더 알고 싶은 부분**: 
> -역전파 알고리즘 수식적인 부분 소화가 안 되었음. chainrule 개념까지는 이해했고, 직접 계산하는 부분에서 직접 손으로 써봐야 할 것 같음. 가중치 역으로 구한다는 것 명확하게 계산해보기. 업데이트식도 명확하게 계산해보기. -> 리뷰 기간 설명가능하도록 소화할 것. (피피티 업데이트하기) 
> -과제에서 feature engineering을 하라고 하는데 이 부분이 어떤 것을 의미하는지 잘 모르겠음. 
> --> 이미지 데이터 차원을 축소하는 것 완벽히 이해하고 사용하는 것 같지 않음. 이 부분을 shape을 변경하는 것 같은데 reshape을 써서 변경하는 부분은 헷갈림. 이걸로 feature engineering을 하는게 맞는지도 모르겠음. CV를 쓰는 것 같기도 한데.. 
> -cifar100 데이터로 keras.add를 써서 cnn모델을 만드는 과제는 거의 진행을 못함. 강의노트에 있는 각각의 하이퍼파라미터를 상세히 살펴보지도 못함. omg....

- **느낀 점**: 
> 재미는 있는데 와우.. 이거 공부할 것이 산더미다. :) (더해보고싶은데쉬겠습니다/오늘은 여기까지:))
